packages/trip-form/src/util.js

```diff
@@ -233,12 +233,12 @@ function getExclusiveModeOptions(ModeIcon, modes, selectedModes) {
 
 /**
  * Generates the options (primary, secondary, tertiary) for the mode selector based on the modes read from config.yaml.
- * @param {*} modes The modes defined in config.yaml.
- * @param {*} icons The icon set to use.
- * @param {*} modes The available modes to choose from.
- * @param {*} selectedModes An array of string that lists the modes selected for a trip query.
- * @param {*} selectedCompanies The companies to show as selected (when the user selects an exclusive mode operated by multiple companies).
- * @param {*} supportedCompanies The supported companies for certain access modes.
+ * @param modes The modes defined in config.yaml.
+ * @param icons The icon set to use.
+ * @param modes The available modes to choose from.
+ * @param selectedModes An array of string that lists the modes selected for a trip query.
+ * @param selectedCompanies The companies to show as selected (when the user selects an exclusive mode operated by multiple companies).
+ * @param supportedCompanies The supported companies for certain access modes.
```

Comment:
These params in the jsdoc don't match the function.

Reply:
Oh my, thanks, I'm a victim of copy-paste!